modules/social_features/social_group/modules/social_group_request/social_group_request.services.yml

```diff
@@ -0,0 +1,8 @@
+services:
+
+  # Defines service for overriding configs
```

Comment:
Move this text to the service class.

Reply:
I'm using official documentation and and I like the style of the example file there: http://prntscr.com/opwwbo